app/src/main/java/com/example/weatherapp/ui/current/ForecastFragment.kt

```diff
@@ -65,18 +64,19 @@ class ForecastFragment : Fragment(), OnMapReadyCallback {
         super.onViewCreated(view, savedInstanceState)
 
         unitSystem = activityViewModel.unitSystem
-        cityName = args.location
-        viewModel.setupData(cityName, unitSystem)
+        viewModel.setupData(args.location, unitSystem)
```

Comment:
Change location argument name to be more descriptive. `cityName` was a good option